tracer/src/Datadog.Trace.Trimming/Datadog.Trace.Trimming.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Version>3.0.0-prerelease</Version>
     <!-- Unconditionally add a prerelease suffix to the package, but keep the rest of the name so we can associate it with the rest of the release -->
```

Comment:
Nit: "Unconditionally" is not longer true

Reply:
welll... if we _always_ have the Prerelease suffix, it _is_ unconditionally right? 🤔